packages/elements/src/components/status-indicator/__styles__/index.ts

```diff
@@ -44,4 +44,10 @@ export const ElStatusIndicator = styled.span`
     color: var(--intent-danger-text);
     outline-color: var(--intent-danger-dark);
   }
+
+  &.shape-tag {
```

Comment:
Can we add `elShapeTag` as an exported class from elements and bump a minor version please?

Reply:
sure!